presto-druid/src/test/java/com/facebook/presto/druid/TestDruidConfig.java

```diff
@@ -28,10 +29,11 @@
     public void testDefaults()
     {
         assertRecordedDefaults(recordDefaults(DruidConfig.class)
-                    .setDruidBrokerUrl(null)
-                    .setDruidCoordinatorUrl(null)
-                    .setDruidSchema("druid")
-                    .setComputePushdownEnabled(false));
+                .setDruidBrokerUrl(null)
+                .setDruidCoordinatorUrl(null)
+                .setDruidSchema("druid")
+                .setComputePushdownEnabled(false)
+                .setHadoopConfiguration((String) null));
```

Comment:
get it. could the following work?
`setHadoopConfiguration("") `

Reply:
it works, thanks!